### DISCUSSION TITLE
From a model, store in two independent tables

### DISCUSSION THREAD

**Discussion by sgonzalez-hiades** [smartsdk/ngsi-timeseries-api#746] [Ideas]
Would it be possible to store the same model in two independent tables with different attributes? 

I am using NGSI-LD and Orion Context Broker v1.4.

I need to store in two independent tables that start data from the same model but have different attributes.

If I use two subscriptions it stores them in the same table.

1 Subscription:

`{ "description": "Store vehicle data", "type": "Subscription", "entities": [ { "type": "Vehicle" } ], "watchedAttributes": ["alternateName", "color", "license_plate", "name"], "notification": { "attributes": ["id", "alternateName", "color", "license_plate", "name"], "format": "normalized", "endpoint": { "uri": "{{qunatumleap}}:8668/v2/notify", "accept": "application/json" } } } `

2 Subscription:

`{ "description": "Store dynamic data", "type": "Subscription", "entities": [ { "type": "Vehicle" } ], "watchedAttributes": ["location", "heading", "speed"], "notification": { "attributes": ["id","location", "heading", "speed"], "format": "normalized", "endpoint": { "uri": "{{quantumleap}}:8668/v2/notify", "accept": "application/json" } } }`

**Comment by c0c0n3** [comment]
Hello @sgonzalez-hiades :-)

> Would it be possible to store the same model in two independent tables with different attributes?

Yes if you can use two different NGSI entity types to group different attributes. E.g. 

```yaml
type: VehicleInfo
name: MyCar
license_plate: CA764ZA
...
```

and 

```yaml
type: VehicleData
location: 41.40338, 2.17403
speed: 50
...
```

No if you'd like to keep all the attributes you in the same `Vehicle` entity. Unfortunately Quantum Leap doesn't support this out of the box---see also #738 about it.

As a workaround, you could still use the same `Vehicle` entity for all the attributes but different `fiware_servicepath`s for each attribute set. That way you'd be able to easily split the data when you query, e.g.

```sql
select name, license_plate from vehicle where fiware_servicepath = '/vehicle/info'
select location, speed from vehicle where fiware_servicepath = '/vehicle/data'
```

Also in this case you'd only need one catch-all Orion subscription for the `Vehicle` entity instead of two.
